spring-cloud-gcp-docs/src/main/asciidoc/datastore.adoc

```diff
@@ -786,7 +786,7 @@ Custom GQL queries can be mapped to repository methods in one of two ways:
 
 Using the `@Query` annotation:
 
-The names of the tags of the GQL correspond to the `@Param` annotated names of the method parameters.
+The names of the initialTags of the GQL correspond to the `@Param` annotated names of the method parameters.
```

Comment:
why is it called initialTags now?

Reply:
unrelated to this PR, but in line 796 we are missing a `"`
`(@Param("trader_name) String traderName)`

Reply:
I think this was a refactor auto-rename that did something wrong., will revert.